inc/deploygroup_dynamicdata.class.php

```diff
@@ -190,27 +190,66 @@ static function getDatas($itemtype, $params, array $forcedisplay=array()) {
     *
     * @since 0.85+1.0
     *
-    * @param object $group PluginFusioninventoryDeployGroup instance
-    * @return array
+    * @param group the group object
+    * @param use_cache retrieve computers_id from cache (computers_id_cache field)
+    * @return an array of computer ids
     */
-   static function getTargetsByGroup(PluginFusioninventoryDeployGroup $group) {
-      $search_params = PluginFusioninventoryDeployGroup::getSearchParamsAsAnArray($group, false,true);
-      if (isset($search_params['metacriteria']) && empty($search_params['metacriteria'])) {
-         unset($search_params['metacriteria']);
-      }
-      $search_params['sort'] = '';
+   static function getTargetsByGroup(PluginFusioninventoryDeployGroup $group, $use_cache = false) {
+      $ids = array();
+
+      if (!$use_cache || !$ids = self::retrieveCache($group)) {
```

Comment:
Not really agree with that, if you call the function with `$use_cache` = 'false' and you have ids in cache, it use cache.
So 2 options:
* if we not use cache, really not use it
* keep like it but remove the param `$use_cache` in the function